packages/web-components/fast-components/src/breadcrumb-item/README.md

```diff
@@ -0,0 +1,4 @@
+# fast-breadcrumb-item
+*BreadcrumbItem* 
```

Comment:
Should this be here?

Reply:
Updated with a description.
"The `fast-breadcrumb-item` component is meant to be used with [breadcrumb](../breadcrumb/README.md)."